core/trino-main/src/main/java/io/trino/execution/scheduler/EventDrivenFaultTolerantQueryScheduler.java

```diff
@@ -1191,35 +1191,39 @@ private StageId getStageId(PlanFragmentId fragmentId)
 
         private void scheduleTasks()
         {
-            long speculativeTasksWaitingForNode = preSchedulingTaskContexts.values().stream()
+            long standardTasksWaitingForNode = preSchedulingTaskContexts.values().stream()
                     .filter(context -> !context.getNodeLease().getNode().isDone())
-                    .filter(context -> context.getExecutionClass() == SPECULATIVE)
+                    .filter(context -> context.getExecutionClass() == STANDARD)
                     .count();
 
-            long nonSpeculativeTasksWaitingForNode = preSchedulingTaskContexts.values().stream()
+            long speculativeTasksWaitingForNode = preSchedulingTaskContexts.values().stream()
```

Comment:
`Keep executionClass explicit in SchedulingQueue and PrioritizedSchedu…` unnecessarily reorders variables

Reply:
extracted